### ISSUE TITLE
Fix request: False positive on UnconditionalOperationOnLogArgument

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#87]
False positive:
```
    public static void call(String to) {
        if (!LOG.isDebugEnabled()) {
            return;
        }
        LOG.debug("green-> {}", shortenActor(to));
    }
```

**Comment by jborgers** [comment]
fixed
